src/configs/regtest.ts

```diff
@@ -1,41 +1,49 @@
-import type { Config } from "src/configs/base";
-import { Explorer, baseConfig, chooseUrl } from "src/configs/base";
+import {
+    AssetType,
+    type Config,
+    ExplorerType,
+    baseConfig,
+    chooseUrl,
+} from "src/configs/base";
 
 const config = {
     ...baseConfig,
     network: "regtest",
     loglevel: "debug",
     apiUrl: {
-        normal: "http://localhost:9001",
+        normal: "http://localhost:9006",
```

Comment:
why?

Reply:
So I changed this back to port `9001` to run against our regular regtest setup (which includes the required backend commit, I carefully checked). Prbly my skill issue, but can't get the quotes to work, just getting a nice `Error` html.

Help a noob out pls 🙏 

<img width="1861" height="1256" alt="Image" src="https://github.com/user-attachments/assets/beb194c5-7c04-417f-a752-90246924a214" />